homu/files/cfg.toml

```diff
@@ -94,8 +108,13 @@ secret = "{{ secrets['web-secret'] }}"
     "unicode-script": {},
     "webrender": {
         "extra_reviewers": [ "nical", "jrmuizel", "Gankro" ],
+        "taskcluster": True,
+        "travis": False,
+        "webrender": True,
```

Comment:
I think this is meant to be `"appveyor": True,`?

Reply:
Oops. Awkward.